sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/uninstall/UninstallSchedulerTest.java

```diff
@@ -262,22 +262,22 @@ public void testAllButDeregisteredPlanCompletes() throws Exception {
         // Starts with a near-empty plan with only the deregistered call incomplete
         Plan plan = getUninstallPlan(uninstallScheduler);
 
-        List<Status> expected = Arrays.asList(Status.PENDING);
+        Collection<Status> expected = Collections.singletonList(Status.PENDING);
```

Comment:
Is `singletonList` strictly required?

Reply:
Yes. It's being compared against the return value of `getStepStatuses()` which is a list. For whatever reason, a List with one element never matches a Set with one element, even when that element is the same.